k8s/base/minio/secret.yaml

```
@@ -0,0 +1,13 @@
+kind: Secret
+metadata:
+  name: minio-secret
+  labels:
+    app: minio
+    component: storage
+type: Opaque
+data:
+  # Base64 encoded values
+  # Default values for development - CHANGE FOR PRODUCTION
+  root-user: bWluaW9hZG1pbg==      # minioadmin
+  root-password: bWluaW9hZG1pbg==  # minioadmin
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)